src/reactor.rs

```diff
@@ -490,7 +480,7 @@ impl<T> Future for Readable<'_, T> {
 
     fn poll(mut self: Pin<&mut Self>, cx: &mut Context<'_>) -> Poll<Self::Output> {
         ready!(Pin::new(&mut self.0).poll(cx))?;
-        log::trace!("readable: fd={}", self.0.handle.source.raw);
+        log::trace!("readable: fd={:?}", &self.0.handle.source.registration);
```

Comment:
I think it would be better to give `Registration` an adjusted `Debug` or `Display` implementation instead of using `fd={:?}`, which seems like it would generate a bit confusing output.

Reply:
Also, using `&` shouldn't be necessary afaik?